### PR TITLE
feat: Release `@fluentui/react-select` as stable

### PR DESCRIPTION
**Pull request by smhigley**
Releasing Select :)

### CODE REVIEW

**Review by micahgodbolt — APPROVED** [review]

**Review by spmonahan — APPROVED** [review]

**Review by spmonahan — APPROVED** [review]
You need to update `index.stories.tsx` by removing "Preview" from the title of the story so Select shows up with the stable components in Storybook.

**Comment by fabricteam** [comment]
## 📊 Bundle size report

<details>
<summary>Unchanged fixtures</summary>

| Package & Exports                                                                                                                                                             |         Size (minified/GZIP) |
| ----------------------------------------------------------------------------------------------------------------------------------------------------------------------------- | ---------------------------: |
| <samp>react-components</samp> <br /> <abbr title='bundle-size/ButtonProviderAndTheme.fixture.js'>react-components: Button, FluentProvider & webLightTheme</abbr>              | `60.308 kB`<br />`16.821 kB` |
| <samp>react-components</samp> <br /> <abbr title='bundle-size/MultipleComponents.fixture.js'>react-components: Accordion, Button, FluentProvider, Image, Menu, Popover</abbr> | `188.08 kB`<br />`52.847 kB` |
| <samp>react-components</samp> <br /> <abbr title='bundle-size/ProviderAndTheme.fixture.js'>react-components: FluentProvider & webLightTheme</abbr>                            | `34.665 kB`<br />`11.427 kB` |
| <samp>react-portal-compat</samp> <br /> <abbr title='bundle-size/PortalCompat.fixture.js'>PortalCompatProvider</abbr>                                                         |   `6.069 kB`<br />`2.053 kB` |
| <samp>react-select</samp> <br /> <abbr title='bundle-size/Select.fixture.js'>Select</abbr>                                                                                    |   `24.14 kB`<br />`8.271 kB` |
| <samp>react-select</samp> <br /> <abbr title='bundle-size/SelectField.fixture.js'>SelectField</abbr>                                                                          |  `33.139 kB`<br />`10.67 kB` |

</details>
<sub>🤖 This report was generated against <a href='https://github.com/microsoft/fluentui/commit/38f8d75d0e279f6fe50fe5647650592cd648d6b2'>38f8d75d0e279f6fe50fe5647650592cd648d6b2</a></sub>
<div id="bundleSizeComment9423"/>

**Comment by fabricteam** [comment]
## [Perf Analysis (`@fluentui/react-components`)](https://github.com/microsoft/fluentui/wiki/Perf-Testing)
<p>No significant results to display.</p><details><summary>All results</summary><p>
  <table>
  <tr>
    <th>Scenario</th>
    <th>Render type</th>
    <th>
      <a href="https://github.com/microsoft/fluentui/wiki/Perf-Testing#why-are-results-listed-in-ticks-instead-of-time-units">Master Ticks</a>
    </th>
    <th>
      <a href="https://github.com/microsoft/fluentui/wiki/Perf-Testing#why-are-results-listed-in-ticks-instead-of-time-units">PR Ticks</a>
    </th>
    <th>Iterations</th>
    <th>Status</th>
  </tr><tr>
            <td>Avatar</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/26491/perf-test-react-components/Avatar-mount_base.html">1274</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/26491/perf-test-react-components/Avatar-mount.html">1335</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>Button</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/26491/perf-test-react-components/Button-mount_base.html">926</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/26491/perf-test-react-components/Button-mount.html">939</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>FluentProvider</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/26491/perf-test-react-components/FluentProvider-mount_base.html">1576</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/26491/perf-test-react-components/FluentProvider-mount.html">1609</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>FluentProviderWithTheme</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/26491/perf-test-react-components/FluentProviderWithTheme-mount_base.html">634</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/26491/perf-test-react-components/FluentProviderWithTheme-mount.html">626</a></td>
            <td>10</td>
            <td></td>
           </tr>
<tr>
            <td>FluentProviderWithTheme</td>
            <td>virtual-rerender</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/26491/perf-test-react-components/FluentProviderWithTheme-virtual-rerender_base.html">597</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/26491/perf-test-react-components/FluentProviderWithTheme-virtual-rerender.html">595</a></td>
            <td>10</td>
            <td></td>
           </tr>
<tr>
            <td>FluentProviderWithTheme</td>
            <td>virtual-rerender-with-unmount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/26491/perf-test-react-components/FluentProviderWithTheme-virtual-rerender-with-unmount_base.html">612</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/26491/perf-test-react-components/FluentProviderWithTheme-virtual-rerender-with-unmount.html">623</a></td>
            <td>10</td>
            <td></td>
           </tr>
<tr>
            <td>InfoButton</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/26491/perf-test-react-components/InfoButton-mount_base.html">575</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/26491/perf-test-react-components/InfoButton-mount.html">506</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>MakeStyles</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/26491/perf-test-react-components/MakeStyles-mount_base.html">1929</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/26491/perf-test-react-components/MakeStyles-mount.html">1875</a></td>
            <td>50000</td>
            <td></td>
           </tr>
<tr>
            <td>Persona</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/26491/perf-test-react-components/Persona-mount_base.html">2990</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/26491/perf-test-react-components/Persona-mount.html">2978</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>SpinButton</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/26491/perf-test-react-components/SpinButton-mount_base.html">2475</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/26491/perf-test-react-components/SpinButton-mount.html">2507</a></td>
            <td>5000</td>
            <td></td>
           </tr></table></p></details>

<div id="perfComment9425"/>

**Comment by fabricteam** [comment]
## 🕵 fluentuiv9 No visual regressions between this PR and main
<div id="vrtCommentfluentuiv9"/>

**Review by khmakoto — APPROVED** [review]

File: change/@fluentui-react-components-fb13e7de-16e0-4fdc-844f-aeba1d747d9a.json [hunk 1]
```diff
@@ -0,0 +1,7 @@
+{
```
Comment (fabricteam) [inline change/@fluentui-react-components-fb13e7de-16e0-4fdc-844f-aeba1d747d9a.json:1]: ## [🕵 fluentuiv8 Open the Visual Regressions report to inspect the 2 screenshots]( https://staticpagerelease.z16.web.core.windows.net/azurestaticapps/vr-fluent/index.html?organization=uifabric&project=fabricpublic&prId=26491&commitId=ecdbf40bbdbd5f7bb28fc0b234911d4c19ca7596&env=prod)

✅ There was 0 screenshots added, 2 screenshots removed, 1043 screenshots unchanged, 0 screenshots with different dimensions and 0 screenshots with visible difference.


  <details><summary><b>unknown</b> 2 screenshots</summary>

|Image Name|Diff(in Pixels)| Image Type
|-|-|-|
|Pivot - Overflow.Tabs - RTL.Narrow - Last tab selected.chromium.png|0| Removed|
|Pivot - Overflow.Tabs - RTL.Narrow - Overflow menu.chromium.png|0| Removed|

</details>


<div id="vrtCommentfluentuiv8"/>

**Comment by fabricteam** [comment]
## [Perf Analysis (`@fluentui/react`)](https://github.com/microsoft/fluentui/wiki/Perf-Testing)

  <table>
  <tr>
    <th>Scenario</th>
    <th>Render type</th>
    <th>
      <a href="https://github.com/microsoft/fluentui/wiki/Perf-Testing#why-are-results-listed-in-ticks-instead-of-time-units">Master Ticks</a>
    </th>
    <th>
      <a href="https://github.com/microsoft/fluentui/wiki/Perf-Testing#why-are-results-listed-in-ticks-instead-of-time-units">PR Ticks</a>
    </th>
    <th>Iterations</th>
    <th>Status</th>
  </tr><tr>
            <td>buttonNative</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/26491/perf-test/buttonNative-mount_base.html">531</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/26491/perf-test/buttonNative-mount.html">971</a></td>
            <td>5000</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/26491/perf-test/buttonNative-mount.regression.txt">Possible regression</a></td>
           </tr></table><details><summary>All results</summary><p>
  <table>
  <tr>
    <th>Scenario</th>
    <th>Render type</th>
    <th>
      <a href="https://github.com/microsoft/fluentui/wiki/Perf-Testing#why-are-results-listed-in-ticks-instead-of-time-units">Master Ticks</a>
    </th>
    <th>
      <a href="https://github.com/microsoft/fluentui/wiki/Perf-Testing#why-are-results-listed-in-ticks-instead-of-time-units">PR Ticks</a>
    </th>
    <th>Iterations</th>
    <th>Status</th>
  </tr><tr>
            <td>BaseButton</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/26491/perf-test/BaseButton-mount_base.html">1161</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/26491/perf-test/BaseButton-mount.html">1131</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>Breadcrumb</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/26491/perf-test/Breadcrumb-mount_base.html">2676</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/26491/perf-test/Breadcrumb-mount.html">2748</a></td>
            <td>1000</td>
            <td></td>
           </tr>
<tr>
            <td>Checkbox</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/26491/perf-test/Checkbox-mount_base.html">2356</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/26491/perf-test/Checkbox-mount.html">2443</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>CheckboxBase</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/26491/perf-test/CheckboxBase-mount_base.html">2255</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/26491/perf-test/CheckboxBase-mount.html">2148</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>ChoiceGroup</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/26491/perf-test/ChoiceGroup-mount_base.html">4293</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/26491/perf-test/ChoiceGroup-mount.html">4362</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>ComboBox</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/26491/perf-test/ComboBox-mount_base.html">1183</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/26491/perf-test/ComboBox-mount.html">1168</a></td>
            <td>1000</td>
            <td></td>
           </tr>
<tr>
            <td>CommandBar</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/26491/perf-test/CommandBar-mount_base.html">10439</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/26491/perf-test/CommandBar-mount.html">9187</a></td>
            <td>1000</td>
            <td></td>
           </tr>
<tr>
            <td>ContextualMenu</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/26491/perf-test/ContextualMenu-mount_base.html">10986</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/26491/perf-test/ContextualMenu-mount.html">10446</a></td>
            <td>1000</td>
            <td></td>
           </tr>
<tr>
            <td>DefaultButton</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/26491/perf-test/DefaultButton-mount_base.html">1213</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/26491/perf-test/DefaultButton-mount.html">1335</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>DetailsRow</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/26491/perf-test/DetailsRow-mount_base.html">3427</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/26491/perf-test/DetailsRow-mount.html">3307</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>DetailsRowFast</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/26491/perf-test/DetailsRowFast-mount_base.html">3526</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/26491/perf-test/DetailsRowFast-mount.html">3234</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>DetailsRowNoStyles</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/26491/perf-test/DetailsRowNoStyles-mount_base.html">3393</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/26491/perf-test/DetailsRowNoStyles-mount.html">2969</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>Dialog</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/26491/perf-test/Dialog-mount_base.html">2979</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/26491/perf-test/Dialog-mount.html">2883</a></td>
            <td>1000</td>
            <td></td>
           </tr>
<tr>
            <td>DocumentCardTitle</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/26491/perf-test/DocumentCardTitle-mount_base.html">498</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/26491/perf-test/DocumentCardTitle-mount.html">586</a></td>
            <td>1000</td>
            <td></td>
           </tr>
<tr>
            <td>Dropdown</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/26491/perf-test/Dropdown-mount_base.html">2965</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/26491/perf-test/Dropdown-mount.html">3117</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>FocusTrapZone</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/26491/perf-test/FocusTrapZone-mount_base.html">1894</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/26491/perf-test/FocusTrapZone-mount.html">1874</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>FocusZone</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/26491/perf-test/FocusZone-mount_base.html">1844</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/26491/perf-test/FocusZone-mount.html">1822</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>GroupedList</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/26491/perf-test/GroupedList-mount_base.html">1895</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/26491/perf-test/GroupedList-mount.html">2057</a></td>
            <td>2</td>
            <td></td>
           </tr>
<tr>
            <td>GroupedList</td>
            <td>virtual-rerender</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/26491/perf-test/GroupedList-virtual-rerender_base.html">1015</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/26491/perf-test/GroupedList-virtual-rerender.html">1056</a></td>
            <td>2</td>
            <td></td>
           </tr>
<tr>
            <td>GroupedList</td>
            <td>virtual-rerender-with-unmount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/26491/perf-test/GroupedList-virtual-rerender-with-unmount_base.html">1627</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/26491/perf-test/GroupedList-virtual-rerender-with-unmount.html">1593</a></td>
            <td>2</td>
            <td></td>
           </tr>
<tr>
            <td>GroupedListV2</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/26491/perf-test/GroupedListV2-mount_base.html">523</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/26491/perf-test/GroupedListV2-mount.html">530</a></td>
            <td>2</td>
            <td></td>
           </tr>
<tr>
            <td>GroupedListV2</td>
            <td>virtual-rerender</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/26491/perf-test/GroupedListV2-virtual-rerender_base.html">541</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/26491/perf-test/GroupedListV2-virtual-rerender.html">504</a></td>
            <td>2</td>
            <td></td>
           </tr>
<tr>
            <td>GroupedListV2</td>
            <td>virtual-rerender-with-unmount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/26491/perf-test/GroupedListV2-virtual-rerender-with-unmount_base.html">567</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/26491/perf-test/GroupedListV2-virtual-rerender-with-unmount.html">553</a></td>
            <td>2</td>
            <td></td>
           </tr>
<tr>
            <td>IconButton</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/26491/perf-test/IconButton-mount_base.html">1702</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/26491/perf-test/IconButton-mount.html">1931</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>Label</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/26491/perf-test/Label-mount_base.html">709</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/26491/perf-test/Label-mount.html">726</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>Layer</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/26491/perf-test/Layer-mount_base.html">4515</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/26491/perf-test/Layer-mount.html">3958</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>Link</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/26491/perf-test/Link-mount_base.html">764</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/26491/perf-test/Link-mount.html">836</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>MenuButton</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/26491/perf-test/MenuButton-mount_base.html">1562</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/26491/perf-test/MenuButton-mount.html">1542</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>MessageBar</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/26491/perf-test/MessageBar-mount_base.html">2084</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/26491/perf-test/MessageBar-mount.html">2079</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>Nav</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/26491/perf-test/Nav-mount_base.html">2963</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/26491/perf-test/Nav-mount.html">3066</a></td>
            <td>1000</td>
            <td></td>
           </tr>
<tr>
            <td>OverflowSet</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/26491/perf-test/OverflowSet-mount_base.html">1300</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/26491/perf-test/OverflowSet-mount.html">1288</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>Panel</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/26491/perf-test/Panel-mount_base.html">2379</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/26491/perf-test/Panel-mount.html">2407</a></td>
            <td>1000</td>
            <td></td>
           </tr>
<tr>
            <td>Persona</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/26491/perf-test/Persona-mount_base.html">1311</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/26491/perf-test/Persona-mount.html">1309</a></td>
            <td>1000</td>
            <td></td>
           </tr>
<tr>
            <td>Pivot</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/26491/perf-test/Pivot-mount_base.html">1630</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/26491/perf-test/Pivot-mount.html">1574</a></td>
            <td>1000</td>
            <td></td>
           </tr>
<tr>
            <td>PrimaryButton</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/26491/perf-test/PrimaryButton-mount_base.html">1421</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/26491/perf-test/PrimaryButton-mount.html">1442</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>Rating</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/26491/perf-test/Rating-mount_base.html">6615</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/26491/perf-test/Rating-mount.html">6773</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>SearchBox</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/26491/perf-test/SearchBox-mount_base.html">1452</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/26491/perf-test/SearchBox-mount.html">1382</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>Shimmer</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/26491/perf-test/Shimmer-mount_base.html">2641</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/26491/perf-test/Shimmer-mount.html">2590</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>Slider</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/26491/perf-test/Slider-mount_base.html">1971</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/26491/perf-test/Slider-mount.html">2022</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>SpinButton</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/26491/perf-test/SpinButton-mount_base.html">4309</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/26491/perf-test/SpinButton-mount.html">4326</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>Spinner</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/26491/perf-test/Spinner-mount_base.html">801</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/26491/perf-test/Spinner-mount.html">718</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>SplitButton</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/26491/perf-test/SplitButton-mount_base.html">2973</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/26491/perf-test/SplitButton-mount.html">2858</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>Stack</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/26491/perf-test/Stack-mount_base.html">720</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/26491/perf-test/Stack-mount.html">728</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>StackWithIntrinsicChildren</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/26491/perf-test/StackWithIntrinsicChildren-mount_base.html">2185</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/26491/perf-test/StackWithIntrinsicChildren-mount.html">2131</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>StackWithTextChildren</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/26491/perf-test/StackWithTextChildren-mount_base.html">4417</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/26491/perf-test/StackWithTextChildren-mount.html">4597</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>SwatchColorPicker</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/26491/perf-test/SwatchColorPicker-mount_base.html">9756</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/26491/perf-test/SwatchColorPicker-mount.html">11600</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>TagPicker</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/26491/perf-test/TagPicker-mount_base.html">2507</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/26491/perf-test/TagPicker-mount.html">2477</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>TeachingBubble</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/26491/perf-test/TeachingBubble-mount_base.html">87626</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/26491/perf-test/TeachingBubble-mount.html">85847</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>Text</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/26491/perf-test/Text-mount_base.html">802</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/26491/perf-test/Text-mount.html">766</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>TextField</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/26491/perf-test/TextField-mount_base.html">1510</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/26491/perf-test/TextField-mount.html">1497</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>ThemeProvider</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/26491/perf-test/ThemeProvider-mount_base.html">1841</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/26491/perf-test/ThemeProvider-mount.html">1469</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>ThemeProvider</td>
            <td>virtual-rerender</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/26491/perf-test/ThemeProvider-virtual-rerender_base.html">1071</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/26491/perf-test/ThemeProvider-virtual-rerender.html">1008</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>ThemeProvider</td>
            <td>virtual-rerender-with-unmount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/26491/perf-test/ThemeProvider-virtual-rerender-with-unmount_base.html">2025</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/26491/perf-test/ThemeProvider-virtual-rerender-with-unmount.html">1991</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>Toggle</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/26491/perf-test/Toggle-mount_base.html">1049</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/26491/perf-test/Toggle-mount.html">997</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>buttonNative</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/26491/perf-test/buttonNative-mount_base.html">531</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/26491/perf-test/buttonNative-mount.html">971</a></td>
            <td>5000</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/26491/perf-test/buttonNative-mount.regression.txt">Possible regression</a></td>
           </tr></table></p></details>

<div id="perfComment9423"/>

**Pull request merged**